lib/jsdom/living/nodes/Document-impl.js

```diff
@@ -589,7 +918,7 @@ class DocumentImpl extends NodeImpl {
   get body() {
     const { documentElement } = this;
     if (!documentElement || documentElement._localName !== "html" ||
-        documentElement._namespaceURI !== HTML_NS) {
+      documentElement._namespaceURI !== HTML_NS) {
```

Comment:
All these reformatting changes are unfortunate :(. If you could undo them that would be really really nice.

Reply:
Yep, sorry about that, I tried to be careful about them, but I missed them.